Libplanet.Tests/Blocks/BlockCommitExtensionsTest.cs

```diff
@@ -11,139 +11,89 @@ namespace Libplanet.Tests.Blocks
-                         hash,
-                         DateTimeOffset.UtcNow,
-                         new PrivateKey().PublicKey,
-                         VoteFlag.Commit).Sign(invalidValidator),
-                 }.ToImmutableArray());
-             Assert.False(emptySignatureCommit.HasValidVotes());
-         }
-
-         [Fact]
-         public void HasVotesSameHeight()
-         {
-             var hash = new BlockHash(TestUtils.GetRandomBytes(32));
-             var sameHeight = new BlockCommit(
-                 2,
-                 0,
-                 hash,
-                 new[]
-                 {
-                     new VoteMetadata(
-                         2,
-                         0,
-                         hash,
-                         DateTimeOffset.UtcNow,
-                         TestUtils.ConsensusPeer0PrivateKey.PublicKey,
-                         VoteFlag.Commit).Sign(TestUtils.ConsensusPeer0PrivateKey),
-                     new VoteMetadata(
-                         2,
-                         0,
-                         hash,
-                         DateTimeOffset.UtcNow,
-                         TestUtils.ConsensusPeer1PrivateKey.PublicKey,
-                         VoteFlag.Commit).Sign(TestUtils.ConsensusPeer1PrivateKey),
-                 }.ToImmutableArray());
-             Assert.True(sameHeight.HasVotesSameHeight());
-             var differentHeight = new BlockCommit(
-                 2,
-                 0,
-                 hash,
-                 new[]
-                 {
-                     new VoteMetadata(
-                         2,
-                         0,
-                         hash,
-                         DateTimeOffset.UtcNow,
-                         TestUtils.ConsensusPeer0PrivateKey.PublicKey,
-                         VoteFlag.Commit).Sign(TestUtils.ConsensusPeer0PrivateKey),
-                     new VoteMetadata(
-                         1,
-                         0,
-                         hash,
-                         DateTimeOffset.UtcNow,
-                         TestUtils.ConsensusPeer1PrivateKey.PublicKey,
-                         VoteFlag.Commit).Sign(TestUtils.ConsensusPeer1PrivateKey),
-                 }.ToImmutableArray());
-             Assert.False(differentHeight.HasVotesSameHeight());
-         }
+            var invalidValidator = new PrivateKey();
+            var emptySignatureCommit = new BlockCommit(
+                2,
+                0,
+                hash,
+                new[]
+                {
+                    new VoteMetadata(
+                        2,
+                        0,
+                        hash,
+                        DateTimeOffset.UtcNow,
+                        new PrivateKey().PublicKey,
+                        VoteFlag.Commit).Sign(invalidValidator),
+                }.ToImmutableArray());
+            Assert.False(emptySignatureCommit.HasValidVotes());
+        }
```

Comment:
Think I named a variable wrong. Can you change the `emptySignatureCommit` to `invalidSignatureCommit`?

Reply:
Done. See aef376dcc8fad67b1679ee746c354d80733534ca.